Makefile

```diff
@@ -1,4 +1,4 @@
-PYTHON ?= python3.8
+PYTHON ?= python
```

Comment:
When we discussed this I was more thinking about `export PYTHON=python` in the YAML. Because if you change it here you'll also have to change the README which currently mentions this:

> Make sure you have the `python3.8` binary in your PATH.

And of course this doesn't apply if the Makefile suddenly looks for a `python` binary.